plugins/modules/fusion_sc.py

```diff
@@ -155,11 +152,11 @@ def create_sc(module, fusion):
         )
     )
     bw_limit = parse_number_with_metric_suffix(module, module.params["bw_limit"])
-    if bw_limit not in range(1048576, 549755813889):  # 1MB/s to 512GB/s
+    if bw_limit < 1048576 or bw_limit > 549755813888:  # 1MB/s to 512GB/s
         module.fail_json(msg="Bandwidth limit is not within the required range")
-    if 100 > iops_limit > 10000000:
+    if iops_limit < 100 or iops_limit > 10000000:
```

Comment:
The iops limit is 100,000,000

Reply:
(as a minor note, Python ignores underscores in numeric literals, so we could use them to separate magnitudes)

Reply:
Thanks for noticing!